imports/plugins/included/authnet/server/methods/authnet.js

```diff
@@ -153,6 +188,18 @@ function priorAuthCaptureTransaction(transId, amount, service) {
   return Promise.await(transactionRequest);
 }
 
+function voidTransaction(transId, service) {
+  let body = {
+    transactionType: "voidTransaction",
+    refTransId: transId
+  };
+  // This call returns a Promise to the cb so we need to use Promise.await
+  let transactionRequest = service.sendTransactionRequest.call(service, body, function (trans) {
+    return trans;
+  });
+  return Promise.await(transactionRequest);
+}
+
 ValidCardNumber = Match.Where(function (x) {
```

Comment:
just a thought for the future, these card validations could perhaps move into the schemas and we validate against schemas rather than this object locally.